src/qt/guiutil.cpp

```diff
@@ -1556,86 +1583,59 @@ void updateFonts()
         if (std::find(vecIgnore.begin(), vecIgnore.end(), w->metaObject()->className()) != vecIgnore.end()) {
             continue;
         }
+        ++nUpdatable;
+
         QFont font = w->font();
+        assert(font.pointSize() > 0);
```

Comment:
I really don't love using asserts... If they do catch they provide basically no info to the user for us to debug with...

Reply:
`>0` means that it's either a default font size (set via `setApplicationFont`) or some another valid value and font size was _not_ set via `setPixelSize`. IMO it's safe to use it here, should only crash if we break smth in code (e.g. use `setPixelSize`).